pkg/config/config_template.yaml

```diff
@@ -276,11 +276,15 @@ metadata_collectors:
 #
 # exclude_pause_container: true
 
-# Use `ignore_docker_cloud_containers` if you are running the agent in Docker Cloud
-# and you want the Docker check to ignore the containers used for the Docker cloud agent.
-# The agent will ignore the docker images starting with `dockercloud/*` (i.e.: dockercloud/network-daemon).
+# Exclude default containers from DockerCloud:
+# The following configuration will instruct the agent to ignore the containers from Docker Cloud.
+# You can remove the ones you want to collect.
+# ac_exclude: ["image:dockercloud/network-daemon","image:dockercloud/cleanup","image:dockercloud/logrotate","dockercloud/events","dockercloud/ntpd"]
```

Comment:
looks like you're missing `image:` in the last two ones

Reply:
🤦‍♂️  thanks for the 👀